### PR TITLE
#49 code coverage

### PR DESCRIPTION
**Pull request by seanjohnite**
Might be pretty much done with the coverage here. I wasn't sure about the best way to mock out the "nets" module, so I did it similarly to how things were mocked out in the "check()" tests. Thoughts?

### CODE REVIEW

File: tests/unit/utils-request.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    .then(function () {
+      simple.restore()
+    })
+})
+
+test('request rejects with error', function (t) {
+  t.plan(4)
+
+  simple.mock(request.internals, 'nets').callFn(function (options, cb) {
+    cb({}, {})
```
Comment (gr2m) [inline tests/unit/utils-request.js]: could you pass in an error as the first argument? I think the 2nd argument shouldn’t be necessary then:

```js
cb(new Error('oops'))
```

**Comment by gr2m** [comment]
A+, well done!

One minor: could you rename `cb` to `callback`? We try to avoid abbreviations if possible.

If you could squash your commits to only one `test: 100% coverage`, then it’s good to go from my side

**Comment by seanjohnite** [comment]
All set I think!

**Comment by finnp** [comment]
Awesome, LGTM! :relaxed:

**Comment by gr2m** [comment]
jap, LGTM :+1:

**Pull request merged**